server/lib/system/system.getGladysBasePath.js

```diff
@@ -11,16 +13,20 @@ async function getGladysBasePath() {
     const base = process.env.SQLITE_FILE_PATH;
     basePathOnContainer = base.substring(0, base.lastIndexOf('/'));
   }
-  // Find mount linked to this path to fetch host path
-  const currentContainerId = await this.getGladysContainerId();
-  const gladysMounts = await this.getContainerMounts(currentContainerId);
-  if (gladysMounts) {
-    const baseMount = gladysMounts.find((mount) => {
-      return mount.Destination === basePathOnContainer;
-    });
-    if (baseMount) {
-      return { basePathOnContainer, basePathOnHost: baseMount.Source };
+  try {
+    // Find mount linked to this path to fetch host path
+    const currentContainerId = await this.getGladysContainerId();
+    const gladysMounts = await this.getContainerMounts(currentContainerId);
+    if (gladysMounts) {
+      const baseMount = gladysMounts.find((mount) => {
+        return mount.Destination === basePathOnContainer;
+      });
+      if (baseMount) {
+        return { basePathOnContainer, basePathOnHost: baseMount.Source };
+      }
     }
+  } catch (e) {
```

Comment:
I'm not sure we want to change this system API in this PR. Other integrations relies on this function already

Reply:
Ok, I have removed this change